multipadic.py

```diff
@@ -1,3 +1,6 @@
+from create_zorgette_catalogue import Tuple
+from create_zorgette_catalogue import Union
+from create_zorgette_catalogue import List
```

Comment:
These are in the wrong place. They should appear *after* the #!/usr/bin/env python3 line